mmv1/products/compute/terraform.yaml

```diff
@@ -176,6 +176,21 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "image_backend"
         vars:
           backend_bucket_name: "image-backend-bucket"
+      - !ruby/object:Provider::Terraform::Examples
```

Comment:
Can we skip the registry docs on this one?

Reply:
sure